uui/settings.ts

```diff
@@ -1,315 +1,1148 @@
+            mobileSearchInput: '48',
+            modalRow: '36',
+            modalPadding: '24',
         },
-        text: {
-            18: { lineHeight: 12, fontSize: 10 },
-            24: { lineHeight: 18, fontSize: 12 },
-            30: { lineHeight: 18, fontSize: 14 },
-            36: { lineHeight: 18, fontSize: 14 },
-            42: { lineHeight: 24, fontSize: 16 },
-            48: { lineHeight: 24, fontSize: 16 },
-            60: { lineHeight: 30, fontSize: 24 },
+    },
+};
+
+interface RadioInputIcons {
+    dotIcon: Icon;
+}
+
+interface RadioInputSizes {
+    default: RadioInputProps['size'];
+}
+
+interface RadioInputSettings {
+    icons: RadioInputIcons;
+    sizes: RadioInputSizes;
+}
+
+const radioInputSettings: RadioInputSettings = {
+    icons: {
+        dotIcon: DotIcon,
+    },
+    sizes: {
+        default: '18',
+    },
+};
+
+interface RangeDatePickerIcons {
+    input: {
+        calendarIcon: Icon;
+    }
+}
+
+interface RangeDatePickerSizes {
+    default: RangeDatePickerProps['size'];
+}
+
+interface RangeDatePickerSettings {
+    icons: RangeDatePickerIcons;
+    sizes: RangeDatePickerSizes;
+}
+
+const rangeDatePickerSettings: RangeDatePickerSettings = {
+    icons: {
+        input: {
+            calendarIcon: CalendarIcon,
         },
-        filtersPanel: {
-            pickerInput: {
-                body: {
-                    default: '36',
-                },
-            },
+    },
+    sizes: {
+        default: '36',
+    },
+};
+
+interface RatingIcons {
+    filledRatingIcon: Icon;
+    emptyRatingIcon: Icon;
+}
+
+interface RatingSizes {
+    default: RatingProps['size'];
+}
+
+interface RatingSettings {
+    icons: RatingIcons;
+    sizes: RatingSizes;
+}
+
+const ratingSettings: RatingSettings = {
+    icons: {
+        filledRatingIcon: FilledStarIcon,
+        emptyRatingIcon: FilledStarIcon,
+    },
+    sizes: {
+        default: 18,
+    },
+};
+
+interface RichTextViewSizes {
+    default: RichTextViewProps['size'];
+}
+
+interface RichTextViewSettings {
+    sizes: RichTextViewSizes;
+}
+
+const richTextViewSettings: RichTextViewSettings = {
+    sizes: {
+        default: '14',
+    },
+};
+
+interface RowAddonsIcons {
+    foldingIcon: Icon;
+}
+
+interface RowAddonsSizes {
+    defaultIcon: number | string;
+    defaultIndentUnit: number;
+    defaultIndentWidth: number;
+    checkboxMap: Sizes<DataRowAddonsProps<unknown, unknown>['size'], CheckboxProps['size']>;
+    iconMap: Sizes<DataRowAddonsProps<unknown, unknown>['size'], number | string>;
+    indentUnitMap: Sizes<DataRowAddonsProps<unknown, unknown>['size'], number>;
+    indentWidthMap: Sizes<DataRowAddonsProps<unknown, unknown>['size'], number>;
+}
+
+interface RowAddonsSettings {
+    icons: RowAddonsIcons;
+    sizes: RowAddonsSizes;
+}
+
+const rowAddonsSettings: RowAddonsSettings = {
+    icons: {
+        foldingIcon: DropdownIcon,
+    },
+    sizes: {
+        defaultIcon: '18',
+        defaultIndentUnit: 24,
```

Comment:
remove this defaults
        defaultIndentUnit: 24,
        defaultIndentWidth: 18,